src/strings.json

```diff
@@ -0,0 +1,23 @@
+
+  {
+    "notRequired": "Not required for your state",
+    "newWindow": "Open form in a new window",
+    "nameChange": "I have legally changed my name since I last registered in this state.",
+    "dob": "Date of Birth",
+    "month": "Month",
+    "day": "Day",
+    "year": "Year",
+    "stateName": "Go to the PDF form on %state_name%'s website",
+    "checkReg": "Check your registration",
+    "learnMore": "Learn more about your voting options",
+    "backBtn": "Back to Vote.gov",
+    "stateOnlineName": "Go to %state_name%'s online form",
```

Comment:
Update this placeholder to be `@state_name` to be consistent.

Reply:
updated to @state_name

Reply:
updated